version.sh

```diff
@@ -135,27 +127,40 @@ echo "# Python: $(which python) [$(python --version)]"
 
 seen_mirgecom=0
 
-for i in "${!module_names[@]}"; do
-    url=${module_full_urls[$i]}
-    name=${module_names[$i]}
-    branch=${module_branches[$i]/--branch /}
-    giturl=${url/\#egg=[a-z]*/}
-    [[ ${url} =~ (#egg=[a-z]*) ]] && egg=${BASH_REMATCH[1]} || egg=""
-    [[ -z $url ]] && continue # Ignore non-Git modules
+for name in */; do
```

Comment:
Package enumeration is now worse (here and elsewhere): There's no authoritative source of information on what the packages managed by emirge actually are.

I think that's a marked downside of this direction.

Reply:
True, but the info was (kind of) inaccurate before as well - switching (e.g.) between mirgecom branches might change requirements.txt, but the packages installed/managed by emirge will not

Reply:
:shrug: I can live with this. I don't love it, but I don't see a better option either. The parsing code was bad, too.